### PR TITLE
Hyunzee/form settings

### PR DESCRIPTION
**Pull request by GodGreg**

### CODE REVIEW

File: src/Components/FormCreation/CreateEditFormHeader.tsx [hunk 1]
```diff
@@ -86,6 +91,9 @@ function CreateEditFormHeader({
           updateHeader();
         }}
       />
+      <Button color="primary" startIcon={<FormSettingsOutlineIcon />}>
+        Form Settngs
```
Comment (ahmedhamodi) [inline src/Components/FormCreation/CreateEditFormHeader.tsx:95]: typo! Also, the button doesn't do anything onClick. Should it open a popup / new page?

**Review by ahmedhamodi — COMMENTED** [review]
When I click the "form settings" button, nothing happens. Not sure if that was supposed to be complete in this PR or not.

**Comment by ahmedhamodi** [comment]
> I think this PR is just a start for the work that Hyunzee was able to get done; it isn't a complete implementation. The ticket is pretty big in scope.

Yup that's fine, was just clarifying. I was expecting a popup or new page, but it looks like you have it under control.

**Comment by xinhaoz** [comment]
This can be reviewed for UI and the full functionality for changing theme colour. The image upload and confirmation message can be finished up in another PR.

**Comment by ahmedhamodi** [comment]
I'll review tomorrow after my exam!

**Review by ahmedhamodi — APPROVED** [review]
Tested locally and this looks **really** cool. Good work @xinhaoz and @hyunzeekim!!

**Comment by ahmedhamodi** [comment]
I'll handle the merge conflicts.

**Comment by GodGreg** [comment]
Rebased with master. @ahmedhamodi You can merge once you confirm it looks good.

**Comment by ahmedhamodi** [comment]
Testing now

**Pull request merged**